src/main/fc/init.c

```diff
@@ -303,7 +307,7 @@ void init(void)
 
     delayMicroseconds(10);  // allow configuration to settle // XXX Could be removed, too?
 
-    if (!isMPUSoftReset()) {
+    if (debugResetEEPROM || !isMPUSoftReset()) {
```

Comment:
This will evaluate to TRUE and start checking buttons (if configured)...

Reply:
Because `isMPUSoftReset()` evaluates to false after a breakpoint? If so, why is  `debugResetEEPROM` needed at all?

Reply:
No, because `debugResetEEPROM` has been set to TRUE when stopped at `init`.